ui-admin/src/study/participants/datarepo/DatasetDashboard.tsx

```diff
@@ -117,6 +117,8 @@ const DatasetDashboard = ({ studyEnvContext }: {studyEnvContext: StudyEnvContext
                   <label>Description:</label> { datasetDetails?.description ?
                     datasetDetails?.description : <span className="fst-italic">N/A</span> }
                   <br/>
+                  <label>Created By:</label> { datasetDetails?.createdBy }
```

Comment:
`label` isn't really appropriate here since there's no form element associated with it. I think `dl` would be the better option for this list.